armi/bookkeeping/visualization/vtk.py

```diff
@@ -0,0 +1,363 @@
+# Copyright 2020 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Visualization implementation for VTK files.
+
+Limitations
+-----------
+This version of the VTK file writer comes with a number of limitations and/or aspects
+that can be improved upon. For instance
+ - Only the Block mesh and related parameters are exported to the VTK file. Adding
+   Assembly and Core meshes is totally doable, and will be the product of future work.
```

Comment:
not to mention components! A super epic later development might be plotting e.g pin-reconstructed fluxes in a whole core.